extensions/wikia/TwitterTag/TwitterTagHooks.class.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+class TwitterTagHooks {
+	
+	/*
+	 * onParserFirstCallInit
+	 *
+	 * Registers the <twitter> tag with the parser and sets its callback
+	 *
+	 * @param $parser - The parser
+	 * @return true
+	 */
+	static public function onParserFirstCallInit( Parser &$parser ) {
+		$parser->setHook( 'twitter', [ __CLASS__, 'parseTwitterTag' ] );
+		return true;
+	}
+	
+	/*
+	 * parseTwitterTag
+	 *
+	 * Parses the twitter tag. Checks to ensure the required attributes are there. 
+	 *   Then constructs the HTML after seeing which attributes are in use. 
+	 *
+	 * @param $input - not used
+	 * @param $args - The attributes to the tag in an assoc array
+	 * @param $parser - not used
+	 * @param $frame - not used
+	 * @return $html - The HTML for the twitter tag
+	 */
+	static public function parseTwitterTag( $input, array $args, Parser $parser, PPFrame $frame ) {
+		if( empty( $args['user'] ) ) {
+			$html = wfMessage( "twittertag-nouser" )->parse();
+			return "<strong class='error'>" . $html . "</strong>";
```

Comment:
The coding convention is to [prefer single quotes unless you're doing string interpolation](https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP#String_literals). So, this (and the one below) should either be single quotes, or double quotes but use interpolation.